micro-rdk/src/esp32/tls.rs

```diff
@@ -309,15 +326,6 @@ impl Esp32TLSStream {
     }
 }
 
-impl Drop for Esp32TLSStream {
-    fn drop(&mut self) {
-        log::error!("dropping the tls stream");
-        if let Some(err) = EspError::from(unsafe { esp_tls_conn_destroy(*self.tls_context) }) {
-            log::error!("error while dropping the tls connection '{}'", err);
```

Comment:
I'm not sure how important these log messages are, but they will no longer happen. If they are valuable then they should migrate somewhere else.

Reply:
I'll bring them back